cmd/meroxa/turbine/ruby/run.go

```diff
@@ -0,0 +1,12 @@
+package turbinerb
+
+import (
+	"context"
+)
+
+const GrpcServerPort = 50500
+
+func (t *turbineRbCLI) Run(ctx context.Context) (err error) {
+	// Run turbine-core gRPC server
+	return err
```

Comment:
this will return nil, so I recommend adding an error

Reply:
Sounds good!